Makefile

```diff
@@ -91,15 +91,16 @@ frontend-build:
 frontend-build-storybook:
 	cd frontend && npm run build-storybook
 
+APP_MODE ?= false
 
 run-backend:
 	@echo "**** Warning: Running with Helm and dynamic-clusters endpoints enabled. ****"
 
 ifeq ($(UNIXSHELL),true)
-	HEADLAMP_BACKEND_TOKEN=headlamp HEADLAMP_CONFIG_ENABLE_HELM=true HEADLAMP_CONFIG_ENABLE_DYNAMIC_CLUSTERS=true ./backend/headlamp-server -dev -proxy-urls https://artifacthub.io/* -enable-dynamic-clusters
```

Comment:
here I removed `-enable-dynamic-clusters` since we were already setting HEADLAMP_CONFIG_ENABLE_DYNAMIC_CLUSTERS=true as an env var